crates/hulk_behavior_simulator/src/bin/penalty_shootout_defending.rs

```diff
@@ -0,0 +1,96 @@
+use bevy::prelude::*;
+
+use linear_algebra::{point, vector, Isometry2, Vector};
+use scenario::scenario;
+use spl_network_messages::{GameState, PlayerNumber, Team};
+use types::{
+    ball_position::SimulatorBallState,
+    motion_command::{JumpDirection, MotionCommand},
+};
+
+use hulk_behavior_simulator::{
+    ball::BallResource,
+    game_controller::{GameController, GameControllerCommand},
+    robot::Robot,
+    time::{Ticks, TicksTime},
+};
+
+#[scenario]
+fn penalty_shootout_defending(app: &mut App) {
+    app.add_systems(Startup, startup);
+    app.add_systems(Update, update);
+}
+fn startup(
+    mut commands: Commands,
+    mut game_controller_commands: EventWriter<GameControllerCommand>,
+    mut ball: ResMut<BallResource>,
+) {
+    let mut robot = Robot::new(PlayerNumber::One);
+    *robot.ground_to_field_mut() = Isometry2::from_parts(vector![-4.5, 0.0], 0.0);
+    commands.spawn(robot);
+    ball.state = Some(SimulatorBallState {
+        position: point!(-3.2, 0.0),
+        velocity: Vector::zeros(),
+    });
+    game_controller_commands.send(GameControllerCommand::SetGamePhase(
+        spl_network_messages::GamePhase::PenaltyShootout {
+            kicking_team: Team::Opponent,
+        },
+    ));
+    game_controller_commands.send(GameControllerCommand::SetGameState(GameState::Set));
+}
+
+#[allow(clippy::too_many_arguments)]
+fn update(
+    game_controller: ResMut<GameController>,
+    time: ResMut<Time<Ticks>>,
+    mut ball: ResMut<BallResource>,
+    mut exit: EventWriter<AppExit>,
+    mut robots: Query<&mut Robot>,
+) {
+    if time.ticks() == 2200 {
+        if let Some(ball) = ball.state.as_mut() {
+            ball.velocity = vector![-2.5, -0.4];
+        }
+    }
+    if time.ticks() > 2200 {
```

Comment:
Why are you waiting over 25 seconds before kicking the ball?

Reply:
I honestly don't have a good answer to that. Probably blindly copied something from another scenario where there is usually a ready phase that takes some time